app/policies/application_policy.rb

```diff
@@ -0,0 +1,73 @@
+class ApplicationPolicy
+  attr_reader :user, :context, :params
```

Comment:
```suggestion
  attr_reader :context, :params
```

Je pense que `:user` n’est plus nécessaire, il est délégué à `:context`.